package/kedro_viz/data_access/managers.py

```diff
@@ -104,7 +113,12 @@ def add_catalog(self, catalog: DataCatalog, pipelines: Dict[str, KedroPipeline])
             pipelines: A dictionary which holds project pipelines
         """
 
-        self.resolve_dataset_factory_patterns(catalog, pipelines)
+        self._is_new_catalog = IS_DATACATALOG_2 and isinstance(
+            catalog, KedroDataCatalog
+        )
```

Comment:
Why use so many different ways to refer to the `KedroDataCatalog`? New catalog, catalog 2... seems to overcomplicate things unnecessarily to me. I'd just use the actual name `KedroDataCatalog` and variations of that e.g. `kedro_datacatalog` eveywhere.

Reply:
Agreed, I will simplify it.